### PR TITLE
Removed explicit sleep in favor of active waiting for the…

### PR DESCRIPTION
**Pull request by jotes**
Hi @vladikoff!
I rewritten travis-ci tests in order to remove explicit waits. After  that, tests are passing without any problems on linux:
https://travis-ci.org/jotes/fxa-local-dev/builds/143208528

Could you review this patch?

### CODE REVIEW

File: _scripts/install_all.sh [hunk 1]
```diff
@@ -13,6 +13,8 @@ git clone https://github.com/mozilla/fxa-auth-db-mysql.git &
 
 git clone https://github.com/mozilla/fxa-auth-mailer.git &
 
+git clone https://github.com/mozilla/fxa-customs-server.git &
```
Comment (jotes) [inline _scripts/install_all.sh]: Whoops, I'll fix this.

Comment (vladikoff) [inline _scripts/install_all.sh]: needs rebase or something else?

Comment (jotes) [inline _scripts/install_all.sh]: Yeah, I forgot about this. Give me a sec!

**Comment by jotes** [comment]
@vladikoff could you look again? I just rebased this PR.

**Comment by vladikoff** [comment]
Thanks so much!

**Pull request merged**